spec/unit/sync-accumulator.spec.js

```diff
@@ -0,0 +1,274 @@
+"use strict";
+import 'source-map-support/register';
+const sdk = require("../..");
```

Comment:
This was a copypasta job from some of the other unit tests which use a mixture. Have updated this file but heads up that it's everywhere.